### PR TITLE
Feature/67955 requests and comments in two distinct tabs

### PR DESCRIPTION
**Pull request by melsawy**

### CODE REVIEW

File: db/migrate/20200223032114_delete_smooch_bot_comments.rb [hunk 1]
```diff
@@ -0,0 +1,5 @@
+class DeleteSmoochBotComments < ActiveRecord::Migration
+  def change
+  	 Rails.cache.write('check:migrate:delete_smooch_bot_comments', Time.now);
```
Comment (caiosba) [inline db/migrate/20200223032114_delete_smooch_bot_comments.rb]: Please fix the indentation and remove the `;` at the end of the line

File: test/models/bot/smooch_3_test.rb [hunk 2]
```diff
@@ -326,10 +329,10 @@ def teardown
       end
       pm = ProjectMedia.last
       assert_no_match /#{@media_url}/, pm.text
-      assert_match /#{@media_url_2}/, pm.comments.last.text
+      # assert_match /#{@media_url_2}/, pm.comments.last.text
       assert_equal 'UploadedImage', pm.media.type
-      assert_match /foo/, pm.comments.last.text
-      assert_match /bar/, pm.comments.last.text
+      # assert_match /foo/, pm.comments.last.text
+      # assert_match /bar/, pm.comments.last.text
```
Comment (caiosba) [inline test/models/bot/smooch_3_test.rb]: Why you commented these lines instead of deleting them?

Comment (melsawy) [reply test/models/bot/smooch_3_test.rb]: Deleted commented lines

File: db/migrate/20200223032114_delete_smooch_bot_comments.rb (hunk 1 above)
Comment (melsawy) [reply db/migrate/20200223032114_delete_smooch_bot_comments.rb]: Done

**Pull request merged**